plugins/modules/ec2_vpc_subnet_info.py

```diff
@@ -76,53 +76,83 @@
     returned: success
     type: complex
     contains:
-        subnet_id:
-            description: The ID of the Subnet.
-            returned: always
-            type: str
         id:
-            description: The ID of the Subnet (for backwards compatibility).
-            returned: always
+            description: Subnet resource id
+            returned: I(state=present)
```

Comment:
Shouldn't be always? I don't think I(state=present) is sensible on an info module.